lib/OpenQA/Worker/Job.pm

```diff
@@ -439,9 +439,12 @@ sub _stop_step_5_upload {
         }
 
         # upload other logs and files
-        for my $file (
-            qw(video.ogv video_time.vtt vars.json serial0 autoinst-log.txt serial_terminal.txt virtio_console.log worker-log.txt)
-          )
+        my @files_to_upload = qw(video.ogv video_time.vtt vars.json serial0 autoinst-log.txt serial_terminal.txt virtio_console.log worker-log.txt);
+        if (defined $self->info->{settings}->{VIRTIO_CONSOLE_NUM} && $self->info->{settings}->{VIRTIO_CONSOLE_NUM} > 1) {
+            # in case second virtio_console defined we want to get its log too
+            push(@files_to_upload, 'virtio_console1.log');
```

Comment:
I avoided to upload `virio_consoleX.log` for reason. As I used it for big amount of data. Maybe we should enable it explicit? WDYT?

If we upload depending on VIRTIO_CONSOLE_NUM I would upload all `virtio_consoleX.log` files if present.

Reply:
this code is even very much os-autoinst centric. Maybe it would be better to supply that list from os-autoinst directly, e.g. in a variable, WDYT?